wmn-data.json

```diff
@@ -5984,21 +5986,16 @@
         "cat" : "blog"
       },
       {
-        "name" : "Virustotal",
-        "uri_check" : "https://www.virustotal.com/ui/users/{account}",
+        "name" : "VirusTotal",
+        "uri_check" : "https://www.virustotal.com/ui/users/{account}/avatar",

```

Comment:
Neither https://www.virustotal.com/gui/user/{account} nor https://www.virustotal.com/ui/users/{account} will work. Accessing this urls requires full browser simulation, such as using Selenium, which is significantly more complex than using curl_cffi. We should either remove this feature or implement an alternative technique, like avatar detection.

Reply:
For now, this project in its current state cannot do avatar/file detection. It may be an option for the future and would require extending the schema. 

Please remove this for now.

Reply:
Let's update it to either [https://www.virustotal.com/gui/user/{account}](https://www.virustotal.com/gui/user/%7Baccount%7D) or [https://www.virustotal.com/ui/users/{account}](https://www.virustotal.com/ui/users/%7Baccount%7D), and include a protection flag that enforces the use of full browser simulation.